app-arch/torcx/files/vendor.json

```diff
@@ -0,0 +1,11 @@
+{
+    "kind": "profile-manifest-v0",
+    "value": {
+        "images": [
+            {
+                "name": "docker",
+                "reference": "17.05.0"
```

Comment:
We probably want to drop the final `.0` in there, or at least symlink this into `17.05` and/or some well known reference like `com.coreos.cl`.

This is part of a larger discussion about user-facing guarantees though, so perhaps we can defer till beta-promotion.

/cc @euank @crawford @philips